lib/elixir/pages/Syntax Reference.md

```diff
@@ -379,8 +379,8 @@ quote do
 end
 #=> {:sigil_m, [], [{:<<>>, [], ["foo"]}, 'abc']}
 ```
-
-If the sigil letter is in uppercase, no interpolation is allowed in the sigil, otherwise its contents may be dynamic. Compare the quotes below for more information:
+If the sigil letter is lowercase interpolation is supported and its contents may be dynamic.
```

Comment:
Let's keep the empty line before this line. I personally also prefer to be explicit by mentioning both the case where the contents can be dynamic as well as the case where they can't.